microsoftgraph/microsoft-graph-explorer-v4
# Fix: Enable promptless logout

## Overview
 Fixes #1777 
 When a user clicks 'sign out' they will be automatically signed out without the need to select which account to sign out from.


### Notes
See documentation guide: 
https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/logout.md#promptless-logout

## Testing Instructions

* Sign in to your account
* Sign out
* Observe prompt that ends your session by alerting you that you have been signed out
